tox.ini

```diff
@@ -32,6 +32,8 @@ commands=
     beacon: pytest {posargs:tests/beacon/}
     # The following test seems to consume a lot of memory. Restricting to 3 processes reduces crashes
     rpc-state-byzantium: pytest -n3 {posargs:tests/trinity/json-fixtures-over-rpc/test_rpc_fixtures.py -k 'GeneralStateTests and not stQuadraticComplexityTest and Byzantium'}
+    # Uncomment the next line + modify test_rpc_fixtures.py when Constantinople is included in the mainnet config
```

Comment:
Old comment can be deleted?

```suggestion
```

Reply:
We can do it in a follow-up. I don't want to do it myself in case I misread the comment.

Reply:
This was an oversight. It shouldn't have been there - oops.

Reply:
Kinda figured, but also didn't want to hold up the release over it.